packages/atomic/src/components/search/facets/atomic-segmented-facet/atomic-segmented-facet.tsx

```diff
@@ -145,7 +145,11 @@ export class AtomicSegmentedFacet
     if (!this.label) {
       return;
     }
-    return <b class="inline-block my-3 mx-2">{this.label}:</b>;
+    return (
+      <b class="inline-block my-3 mx-2" part="label">
```

Comment:
Btw all parts have to be documented at the top of the components (all parts from the functional components should be documented there as well)

Reply:
Yes !

Reply:
value-box, value-label, value-count, etc. as well